dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 coveralls==3.3.1
-eralchemy==1.5.0
+eralchemy2==1.4.1
```

Comment:
eralchemy is no longer maintained, so I switched to eralchemy2: https://github.com/maurerle/eralchemy2